src/main.rs

```diff
@@ -119,6 +132,18 @@ pub enum ShowCalls {
     All,
 }
 
+impl ShowCalls {
+    pub fn from_str(s: &str) -> Option<Self> {
+        match s {
```

Comment:
I'd do a to_lowercase() -- so that we also accept when people send 'none' or 'all'

Reply:
Ah good call